controllers/tab/workflow/PKPWorkflowTabHandler.inc.php

```diff
@@ -96,22 +99,25 @@ function fetchTab($args, $request) {
 					$dispatcher = $request->getDispatcher();
 
 					import('lib.pkp.classes.linkAction.request.AjaxModal');
+					$submissionId = $submission->getId();
+					$url = $dispatcher->url(
+						$request, ROUTE_COMPONENT, null,
+						'modals.editorDecision.EditorDecisionHandler',
+						'newReviewRound', null, array(
+							'submissionId' => $submissionId,
+							'decision' => SUBMISSION_EDITOR_DECISION_RESUBMIT,
+							'stageId' => $selectedStageId,
+							'reviewRoundId' => $lastReviewRoundId
+						)
+					);
+					$ajaxModal = new AjaxModal(
+						$url,
+						__('editor.submission.newRound'),
+						'modal_add_item'
+					);
 					$newRoundAction = new LinkAction(
 						'newRound',
-						new AjaxModal(
-							$dispatcher->url(
-								$request, ROUTE_COMPONENT, null,
-								'modals.editorDecision.EditorDecisionHandler',
-								'newReviewRound', null, array(
-									'submissionId' => $submission->getId(),
-									'decision' => SUBMISSION_EDITOR_DECISION_RESUBMIT,
-									'stageId' => $selectedStageId,
-									'reviewRoundId' => $lastReviewRoundId
-								)
-							),
-							__('editor.submission.newRound'),
-							'modal_add_item'
-						),
+						$ajaxModal,
```

Comment:
I think this change is no longer needed as there's no reference in the LinkAction constructor's `$actionRequest` parameter.

Reply:
That's right, this change can be ignored and reverted on our side.